src/Webform.js

```diff
@@ -15,6 +15,7 @@ import {
     getArrayFromComponentPath,
 } from "./utils/utils";
 import { eachComponent } from "./utils/formUtils";
+import dragula from "dragula";
```

Comment:
I really do not like that we are now bringing in a rather large library for the Form rendering. I am willing to accept this PR as is, but I would like to have a ticket to "lazy load" dragula if a form that requires re-ordering is rendered.

Reply:
https://formio.atlassian.net/browse/FIO-8656
A ticket has been created to lazy load dragula